libloki/libloki-protocol.js

```diff
@@ -141,9 +141,7 @@
         log.info('empty message sent successfully');
       }
     };
-    const options = {
-      preKeyBundleType: textsecure.protobuf.PreKeyBundleMessage.Type.FRIEND_REQUEST_ACCEPT,
-    };
+    const options = {};
```

Comment:
```suggestion
```
You can also remove the param in the function below.
```
 const outgoingMessage = new textsecure.OutgoingMessage(
      null, // server
       Date.now(), // timestamp,
       [pubKey], // numbers
       content, // message
       true, // silent
       callback, // callback
);
```

Reply:
I thought it may be more explicit to leave the empty object here if we want to pass options in the future? Just trying to minimise the obscurity of this code base haha

Reply:
In the original signal code it was like that since it defaults the options to `{}` if it doesn't get passed in